### ISSUE TITLE
Docker version requirement is wrong

### ISSUE THREAD

**Issue by yujuhong** [GoogleCloudPlatform/kubernetes#4394]
The docker version requirement for starting a local cluster is documented [here] (https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/locally.md) as `1.0.0+`.

I had docker 1.0.x installed on my machine and couldn't start a cluster. After examining the log, the health check sent to the nodes all failed. Manually sending the check showed that the docker version was too old. 
```
curl -L http://127.0.0.1:10250/healthz
Docker version is too old (1.12)
```

**Comment by brendandburns** [comment]
We should update docs.  We require Docker 1.3.

The docker version requirement for starting a local cluster is documented
here
<https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/locally.md>
as 1.0.0+.

I had docker 1.0.x installed on my machine and couldn't start a cluster.
After examining the log, the health check sent to the nodes all failed.
Manually sending the check showed that the docker version was too old.

curl -L http://127.0.0.1:10250/healthz
Docker version is too old (1.12)

—
Reply to this email directly or view it on GitHub
<https://github.com/GoogleCloudPlatform/kubernetes/issues/4394>.

**Comment by saad-ali** [comment]
Closed with #4398
